src/pages/guide/css/themes.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Include CSS | Commerce Frontend Development
+description: Follow these best practices for structuring stylesheets in Adobe Commerce and Magento Open Source themes.
+---
+
+# Include CSS
+
+In the application, CSS files are included in [layout files](../layouts/index.md).
+
+Technically there is an option to include them in template files, but we strongly recommend avoiding this.
+
+<InlineAlert variant="info" slots="text"/>
+
+The CSS class names can be assigned in both templates and layouts.
+
+This topic describes how stylesheets are located by default in the application file system, and the recommended way to include CSS files in layouts.
+
+## How stylesheet files are organized
+
+Conventionally, CSS and Less files are stored only in themes. [Module](https://glossary.magento.com/module) directories do not contain any default styles.
+
+In a [theme directory][], stylesheets are stored in the following locations:
+
+<table>
```

Comment:
This table is janked.

Reply:
Ah, I missed that one. Yeah, converting to a markdown table should fix it.

Reply:
Not very pretty, but better:

![Screen Shot 2022-06-23 at 10 29 45 AM](https://user-images.githubusercontent.com/13662379/175337179-57eb98a0-4116-45ec-bf72-62c39740b815.png)